lib/awspec/type/dynamodb.rb

```diff
@@ -0,0 +1,38 @@
+module Awspec::Type
+  class Dynamodb < ResourceBase
```

Comment:
Could you append `aws_resource Aws::DynamoDB::Table` ?
( So this resource type can unlock advanced use like [ec2](https://github.com/k1LoW/awspec/blob/master/doc/resource_types.md#unlock-advanced-use) )